tests/test_models_user.py

```diff
@@ -0,0 +1,113 @@
+from nose.tools import *
+
+from framework import auth
+from framework.auth import exceptions
+from website import models
+from tests import base
+from tests import factories
+
+
+class UserTestCase(base.OsfTestCase):
```

Comment:
We might want want to move our existing `User` model tests into this file. Also, I like the idea of breaking up our test modules by model like you've done here. Maybe use a new directory: `tests/test_models/test_user.py`.

This can be deferred.